src/plugins/intel_cpu/src/nodes/dft.cpp

```diff
@@ -180,8 +530,12 @@ void gatherToBufferND(float* buffer, const float* data, size_t axis, const std::
     }
 }
 
-void applyBufferND(const float* buffer, float* output, size_t axis, const std::vector<size_t>& dimIndexes,
-                                  const std::vector<size_t>& shape, const std::vector<size_t>& strides) {
+void applyBufferND(const float* buffer,
+                   float* output,
+                   size_t axis,
+                   const std::vector<size_t>& dimIndexes,
+                   const std::vector<size_t>& shape,
+                   const std::vector<size_t>& strides) {
```

Comment:
can we keep the code format align with previous version?(e.g. looks like the maximum code line length changed here)